### PR TITLE
doc: update security annouce for binaries ready

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]

File: locale/en/blog/vulnerability/february-2023-security-releases.md [hunk 1]
```diff
@@ -1,12 +1,100 @@
...
+CRLF injection in the 'host' header potentially allowing attacks such as HTTP response splitting
+and HTTP header injection.
+
+Thank you, to Zhipeng Zhang ([@timon8](https://hackerone.com/timon8)) for reporting this vulnerability and thank you Robert Nagy for fixing it.
+
+Impacts:
+
+* All versions of the 19.x, 18.x and 16.x release lines.
+
+##  Regular Expression Denial of Service in Headers in Node.js fetch API(Low) ([CVE-2023-24807](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-24807))
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/february-2023-security-releases.md]:
```suggestion
## Regular Expression Denial of Service in Headers in Node.js fetch API(Low) ([CVE-2023-24807](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-24807))
```

File: locale/en/blog/vulnerability/february-2023-security-releases.md [hunk 2]
```diff
@@ -1,12 +1,100 @@
...
+The Headers.set() and Headers.append() methods in the
+[fetch API](https://nodejs.org/dist/latest/docs/api/globals.html#fetch) in Node.js where vulnerable to Regular a  Expression Denial of Service (ReDoS) attacks.
+
+Thank you, to Carter Snook for reporting this vulnerability and thank you Rich Trott for fixing it.
+
+Impacts:
+
+* All versions of the 19.x, 18.x, and 16.x release lines.
+
+##  Node.js insecure loading of ICU data through ICU_DATA environment variable (Low) ([CVE-2023-23920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-23920))
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/february-2023-security-releases.md]:
```suggestion
## Node.js insecure loading of ICU data through ICU_DATA environment variable (Low) ([CVE-2023-23920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-23920))
```

File: locale/en/blog/vulnerability/february-2023-security-releases.md [hunk 3]
```diff
@@ -1,12 +1,100 @@
 ---
-date: 2023-02-14T14:00:00.000Z
+date: 2023-02-16T21:00:00.000Z
 category: vulnerability
-title: Tuesday February 14 2023 Security Releases
+title: Tuesday February 16 2023 Security Releases
```
Comment (richardlau) [inline locale/en/blog/vulnerability/february-2023-security-releases.md]:
```suggestion
title: Thursday February 16 2023 Security Releases
```

**Comment by richardlau** [comment]
Should we update the banner as well?

**Comment by mhdawson** [comment]
@richardlau added update for banner.

**Review by richardlau — APPROVED** [review]

**Pull request merged**